.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+	".": "1.4.0"
```

Comment:
@Ndiritu are we expecting this version to be misaligned with the rest?

Reply:
@baywet this would have been bumped by a Release-Please PR on this repo. However, since we're doing the tagging and releases via GitHub Actions, I'll remove Release-Please configs from the target repos to prevent duplicate automations.